### PR TITLE
[skip ci] operations-and-maintenance

### PR DESCRIPTION
**Pull request by goddammit**
some of it is in mods:/ sorry

### CODE REVIEW

File: doc/operation-and-maintenance/Logging-&-monitoring.md [hunk 1]
```diff
@@ -46,38 +47,37 @@ For more information see: [WombatOAM](https://www.erlang-solutions.com/products/
...
 For more details, please visit the Exometer's project page: [Exometer](https://github.com/Feuerlabs/exometer).
 
 **Note that we are using the 1.2.1 version.**
 
-Below you can find sample configuration, it setups graphite reporter which connects
-to graphite running on localhost.
+Below you can find a sample configuration. 
+It sets up a reporter which connects to graphite running on localhost.
 
 You can see an additional option not listed in the Exometer docs - `mongooseim_report_interval`.
```
Comment (michalwski) [inline doc/operation-and-maintenance/Logging-&-monitoring.md]: I think it should be 'You can `set`...'

File: doc/operation-and-maintenance/Mongoose-metrics.md [hunk 2]
```diff
@@ -14,28 +17,29 @@ All metrics are divided into following groups:
...
 | disco_local_features | spiral | host, hook | |
 | disco_local_identity | spiral | host, hook | |
 | disco_local_items | spiral | host, hook | |
 | disco_sm_features | spiral | host, hook | |
 | disco_sm_identity | spiral | host, hook | |
 | disco_sm_items | spiral | host, hook | |
-| host_config_update | spiral | host, hook | Hook run when config update is being performed. Currently this is run only if there is a change in ldap configuraiton |
-| local_send_to_resource_hook | spiral | host, hook | Hook run when a stanza is addressed to server's resource (f.e. localhost/watchdogs) |
+| host_config_update | spiral | host, hook | Hook run when a config update is being performed. 
+Currently this is run only if there is a change in the ldap configuraiton |
```
Comment (michalwski) [inline doc/operation-and-maintenance/Mongoose-metrics.md]: This breaks the table. Did you look at the rendered version on Github?

File: doc/operation-and-maintenance/Mongoose-metrics.md [hunk 3]
```diff
@@ -102,42 +106,45 @@ All metrics are divided into following groups:
...
 
 | Metric name | Type | Description |
 | ----------- | ---- | ----------- |
-| [xmpp,received,xml_stanza_size] | histogram | Received (by the server) stanza size stats after possible decryption and decompression |
-| [xmpp,sent,xml_stanza_size] | histogra | Sent (to the client) stanza size stats before possible encryption and compression |
-| [xmpp,received,compressed_size] | histogram | Received compressed stanza size stats |
-| [xmpp,sent,compressed_size] | histogram | Sent compressed stanza size stats |
-| [xmpp,received,encrypted_size] | histogram | Received encrypted stanza size stats |
-| [xmpp,sent,encrypted_size] | histogram | Sent encrypted stanza size stats|
+| [xmpp,received,xml_stanza_size] | histogram | Size stats of a stanza received by the server after a possible decryption and decompression |
```
Comment (michalwski) [inline doc/operation-and-maintenance/Mongoose-metrics.md]: Maybe this is enough:
`Size stats of a received plain stanza` and similar for the other side.

**Comment by michalwski** [comment]
Also, the base branch of this PR should be `master` and not `mods-review`

**Comment by goddammit** [comment]
should be ok now:)

**Comment by michalwski** [comment]
Thanks!

**Pull request merged**